database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -91,12 +94,16 @@ private String getJDBCPluginId() {
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     super.configurePipeline(pipelineConfigurer);
+    StageConfigurer configurer = pipelineConfigurer.getStageConfigurer();
     DBUtils.validateJDBCPluginPipeline(pipelineConfigurer, dbSinkConfig, getJDBCPluginId());
```

Comment:
does this need to be updated too?

Reply:
yes that has been updated in the PR as well. Since it takes `pipelineConfigurer` we do not need to pass `FailureCollector` explicitly.